docs/user-guide/home-page.md

```diff
@@ -0,0 +1,78 @@
+---
+sidebar-position: 2
+---
+
+# Home page
+
+To get started with Resuminator, you will be greeted with our Home page, which shows you the following screen:
+
+![Home Page](/doc_img/home_page.png)
+
+The Home page can be divided into the following various parts to understand it better ~
+
+## Side-Bar
+
+### Workspace
+
+- **Settings**: Settings tab routes you to your account settings which contain the general, account and advanced tabs.
+
+### Resources
+
+- **Broadmaps**: Broadmaps was a 36 issue weekly newsletter with high-signal information on products, tech, and growth
+  ⚡ Curated with ♥ by [@\_viveknigam\_](https://github.com/viveknigam3003) &
+  [@\_mercurybuddy](https://github.com/merrcury)
+
+- **Help & Support**: This tab routes you to the [Help & Support](https://docs.resuminator.in/docs/support/)
+  page on the Resuminator Docs website for know-how of our product.
+
+- **Discord Server**: Join our [Discord](https://discord.com/invite/uJRFbDHHAJ)
+  for support, feedback and sneak-peaks in our alpha-program.
+
+## Main Space
+
+This the main area where you can add and delete resumes and edit their name / icon by clicking on the name.
+
+### From Scratch
+
+> If you are creating a resume for the first time, you should refer to our
```

Comment:
Can we show this as a normal text?

Reply:
I had the thought of highlighting this text in case someone doesn't know how to create a resume and route them to the quick start guide

Reply:
Someone is coming directly to this section and this section only holds this highlight, instead of this normal text serving purpose better. I pick highlights as more TL;DR version of things